### PR TITLE
[NEW] storybook

### PR DESCRIPTION
**Pull request by ggazzo**
its not a big deal, but that's it.
closes https://github.com/RocketChat/Rocket.Chat.Livechat/issues/1
image

### CODE REVIEW

File: stories/index.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import { h } from 'preact';
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+storiesOf('Button', module).add('render button', () => <button onClick={action('clicked')}>Hello, Preact World!</button>);
```
Comment (sampaiodiego) [inline stories/index.js]: where should this `action('clicked')` do?

**Comment by renatobecker** [comment]
@sampaiodiego and @ggazzo,

Will we have a `develop` branch or will we merge into `master` branch?

**Comment by sampaiodiego** [comment]
I don't think we need a `develop` branch here.. not for now at least.

File: .storybook/webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+console.log('adsasd')
```
Comment (tassoevan) [inline .storybook/webpack.config.js]: @ggazzo I think you forgot something.

**Pull request merged**